crates/web-client/src/import.rs

```diff
@@ -29,22 +29,21 @@ impl WebClient {
                 .map_err(|err| err.to_string())?;
             let account_id = account_data.account.id().to_string();
 
+            let message = match client
+                .add_account(&account_data.account, account_data.account_seed, false)
+                .await
+            {
+                Ok(_) => format!("Imported account with ID: {account_id}"),
+                Err(err) => return Err(js_error_with_context(err, "failed to import account")),
+            };
```

Comment:
I would just map the error here and write the message on line 46.
```suggestion
            client
                .add_account(&account_data.account, account_data.account_seed, false)
                .await
                .map_err(js_error_with_context(err, "failed to import account"))?;
```

Same goes for the other functions here